subsys/kafka/src/main/conf/conf.d/kafka.conf

```diff
@@ -0,0 +1,14 @@
+[kafka]
+# By default, this is disabled
+#
+#enabled=false
+
+bootstrap.servers=
+security.protocol=SSL
+ssl.keystore.location=kafka.client.jks
+ssl.keystore.password=pass
+ssl.key.password=pass
```

Comment:
Not sure if this password will be used in the future. If answer is yes, it should not be recorded here in a config file. Recommended way is to use openshift secret config. One example is ISPNDBConfiguration, which uses system properties to import the secrets(this part is done in indy.sh start script)

Reply:
Yeah, the passwords should not be here as plain text. I copied it somewhere from PNC code but obviously they are not done with it. I will look at the openshift secrets. Thanks for commenting.